plugins/cindy3d/src/js/Ops3D.js

```diff
@@ -362,17 +362,40 @@ CindyJS.registerPlugin(1, "Cindy3D", function(api) {
     let colortype = "pervertex";
     let topology = "open";
     let colors = null;
+    let uv = null;
+    let texture = null;
     let appearance = handleModifsAppearance(
       currentInstance.surfaceAppearance, modifs, {
         "normaltype": (a => normaltype =
                        coerce.toString(a, normaltype).toLowerCase()),
         "colortype": (a => colortype =
                        coerce.toString(a, colortype).toLowerCase()),
         "topology": (a => topology =
-                       coerce.toString(a, normaltype).toLowerCase()),
-        "colors": (a => colors = coerce.toList(a).map(elt => coerce.toColor(elt))),
+                       coerce.toString(a, topology).toLowerCase()),
+        "colors": (a => colors = coerce.toList(a).map(
+          elt => coerce.toColor(elt))),
+        "uv": (a => uv = coerce.toList(a).map(
+          elt => coerce.toHomog(elt, [0, 0, 0], 2))),
+        "texture": (a => texture = coerce.toString(a)),
```

Comment:
With this it is only possible to have textures that are refered as string. But not `cameravideo()`, for instance.
But that could be considered as an additional feature.